Firestore/Source/API/FIRAggregateField.mm

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRAggregateField.h"
+#import "FIRAggregateField+Internal.h"
+#import "FIRFieldPath+Internal.h"
+#import "Firestore/core/src/model/aggregate_field.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+#pragma mark - FIRAggregateField
+
+@interface FIRAggregateField ()
+@property(nonatomic, strong, readwrite) FIRFieldPath *_fieldPath;
+@end
+
+@implementation FIRAggregateField
+- (instancetype)initWithFieldPath:(FIRFieldPath *)fieldPath {
+  if (self = [super init]) {
+    self._fieldPath = fieldPath;
+  }
+  return self;
+}
+
+- (instancetype)initPrivate {
+  if (self = [super init]) {
+  }
+  return self;
+}
```

Comment:
Are these `initPrivate` methods necessary? It looks like something that `init` would automatically do?